tests/test_soak.py

```diff
@@ -38,7 +38,9 @@
                  "--keytab-secret-path", "/{}".format(HDFS_KEYTAB_SECRET),
                  "--conf", "spark.mesos.driverEnv.SPARK_USER=root"] # run as root on soak (centos)
```

Comment:
Just a better practise in python land, using an integer explicitly to unpack the format tuple.
```"--keytab-secret-path", "/{0}".format(HDFS_KEYTAB_SECRET),```

Reply:
Otherwise LGTM

Reply:
One general observation though, the sdk dependencies we are pulling in, shouldn't they be refactored as libraries and stay in JFrog and then pulled in via pip?

Reply:
Also one last observation, since the integration tests are all written in python it might be worthwhile to have a python module that wraps the dcos cli commands and can be called as functions. This can also be a pipable dependency.

Reply:
> One general observation though, the sdk dependencies we are pulling in, shouldn't they be refactored as libraries and stay in JFrog and then pulled in via pip?

This is what we were discussing in our team meetings last week -- especially the ones related to splitting the repo and consuming the SDK resources.

> Also one last observation, since the integration tests are all written in python it might be worthwhile to have a python module that wraps the dcos cli commands and can be called as functions. This can also be a pipable dependency.

There was a dcos cli python library, but it was deprecated and is unsupported. We have found it to be somewhat more stable to invoke the cli directly.